include/init.h

```diff
@@ -70,6 +70,31 @@ void z_sys_init_run_level(int32_t level);
  *
```

Comment:
So Z_* as a prefix to a macro prevents its documentation to be included?

That said, it looks like SYS_INIT() documentation is not present either...

Reply:
It's added by the changes in this PR.

Reply:
ok no other comments then